src/test/resources/feature/3-destinatario/personaFisica/visualizzaNotifiche/VisualizzaNotificaConSoloUnElementoInTimelinePF.feature

```diff
@@ -0,0 +1,67 @@
+
+  @TestSuite
+  @TA_PFDettaglioNotificheConUnElementoInTimeline
+  @PF
+  @visualizzazioneNotifichePF
+
+  @VisualizzaNotificaMonoDestinatarioConUnElementoInTimelinePF
+  Scenario: [TA-FE VISUALIZZAZIONE DETTAGLI DI NOTIFICA PF] - PF  Verifica presenza solo un elemento in timeline contenente la ricevuta di postalizzazione in formato zip - OK_AR_ZIP
+    Given PF - Si effettua la login tramite token exchange come "delegante", e viene visualizzata la dashboard
+    When Si inizializzano i dati per la notifica
+      | modello         | A/R                |
+      | documenti       | 1                  |
+      | oggettoNotifica | Pagamento rata IMU |
+      | costiNotifica   | false              |
+    And Si aggiunge un destinatario alla notifica
+      | indirizzo        | @OK_AR_ZIP       |
+      | codicePostale    | 20147            |
+      | comune           | Milano           |
+      | dettagliComune   | Milano           |
+      | provincia        | MI               |
+      | stato            | Italia           |
+      | nomeCognome      | Gulio Cesare     |
+      | codiceFiscale    | CSRGGL44L13H501E |
+      | tipoDestinatario | PF               |
+    Then Creo in background una notifica per destinatario tramite API REST
+    And Aspetta 5 secondi
+    And Cliccare sulla notifica restituita
+    And Aspetta 200 secondi
+    And Si visualizza correttamente la section Dettaglio Notifica
+    And controllo link per scaricare zip e scarico file Ricevuta di consegna
+    And Aspetta 5 secondi
+    And estraggo il file zip
+    And Controllo sia presente documento pdf
+    And Si elimina file estratto
+    And Logout da portale persona fisica
+
+
+  @VisualizzaNotificaMonoDestinatarioConUnElementoInTimeline890PF
+  Scenario: [TA-FE VISUALIZZAZIONE DETTAGLI DI NOTIFICA PF] - PF  Verifica presenza solo un elemento in timeline contenente la ricevuta di postalizzazione in formato zip - OK_AR_ZIP
+    Given PF - Si effettua la login tramite token exchange come "delegante", e viene visualizzata la dashboard
+    When Si inizializzano i dati per la notifica
+      | modello         | 890                |
+      | documenti       | 1                  |
+      | oggettoNotifica | Pagamento rata IMU |
+      | costiNotifica   | false              |
+    And Si aggiunge un destinatario alla notifica
+      | indirizzo        | @OK_890_ZIP      |
+      | codicePostale    | 20147            |
+      | comune           | Milano           |
+      | dettagliComune   | Milano           |
+      | provincia        | MI               |
+      | stato            | Italia           |
+      | nomeCognome      | Gulio Cesare     |
+      | codiceFiscale    | CSRGGL44L13H501E |
+      | tipoDestinatario | PF               |
+    Then Creo in background una notifica per destinatario tramite API REST
+    And Aspetta 5 secondi
+    And Cliccare sulla notifica restituita
+    And Aspetta 200 secondi
```

Comment:
delete this wait, use polling for status

Reply:
I dont need polling with status, because ricevuta file is created approximately in 3 minutes, without changing the status